index.js

```diff
@@ -20,25 +20,23 @@ const jobs = {
 }
 
 async function main() {
-  const user = await sdk.user().catch(err => sdk.log(err))
-  const person = user && user.me ? `, ${user.me.username}` : ' there'
-  const greeting = `\n👋  Welcome to Incident.sh  👋\n\nHi${person}!\n`
+  const greeting = `\n👋  Welcome to Incident.sh  👋\n\nHi there!\n`
```

Comment:
I don't like this message, maybe drop the `Hi there` part to declutter as a start for now